AUTHORS

```diff
@@ -5,18 +5,20 @@
   Led by Prof. Natasha Sharygina <natasha.sharygina@usi.ch>
 --------------------------------------------------------------
 
-Project leader and main code designer
+Project leaders and main code designers
 -------------------------------------
-Dr. Antti Hyvarinen           <antti.hyvaerinen@usi.ch>
+Dr. Tomas Kolarik           <tomas.kolarik@usi.ch>  
+Dr. Martin Blicha           <martin.blicha@usi.ch>
 
-Previous project leader and main code designer
+Previous project leaders and main code designers
 -------------------------------------
+Dr. Antti Hyvarinen          <antti.hyvaerinen@usi.ch>  
 Dr. Roberto Bruttomesso      <roberto.bruttomesso@usi.ch>
 
 Previous members
 ----------------
-Dr. Aliaksei Tsitovich       <aliaksei.tsitovich@usi.ch>
-Dr. Simone Fulvio Rollini    <simone.fulvio.rollini@usi.ch>
+Dr. Aliaksei Tsitovich       <aliaksei.tsitovich@usi.ch>  
+Dr. Simone Fulvio Rollini    <simone.fulvio.rollini@usi.ch>  
```

Comment:
Is this adding extra space at the end of the line?
Probably should not do that, no?

Reply:
It was my intention, this syntax tells a Markdown renderer to add linebreak after such lines (otherwise it does not). Although it is just a text file it can still be nicely rendered this way ... I admit that the syntax is weird in general, and that Markdown is just not necessary here.

Reply:
I don't think this was intended for Markdown.
But let's not spend too much time on this.
I would prefer without the spaces, but I am fine with this version as well.